src/game/game.service.js

```diff
@@ -398,17 +420,16 @@ angular.module('ngGo.Game.Service', [
      * Clone this game
      */
     Game.prototype.clone = function() {
-
-      //Create new kifu object and get properties
+      // Create a new object and get properties
       var clone = new Game();
       var props = Object.getOwnPropertyNames(this);
 
-      //Copy all properties
+      // Copy all properties
```

Comment:
Can you undo all the added spaces before `//`? I don't like that style of comments and all comments in the existing codebase are without space. Probably a linter setting.